content/doc/book/installing/_master-agent-machines-setup-windows.adoc

```diff
@@ -0,0 +1,61 @@
+[[master-agent-setup-windows]]
+== Windows Agents
+
+Note: There is no need to install Jenkins on the agent machine.
+
+. On your master machine go to *Manage Jenkins* > *Manage Nodes*. +
+image:/images/installing/ManageNodes.png[Manage nodes,width=80%]
+
+. *New Node*
+.. *Enter Node Name*
+.. *Select Permanent Agent*
+.. Press *OK*. +
+image:/images/installing/NewNode.png[New node,width=40%]
```

Comment:
Other images are placed in the `/images/` directory so I placed the images for this pull request there also.

Also, the `image` name has been replaced with a description of the content of the image so that screen readers have the help of the image title to hint what is contained in the image.

Reply:
Sorry for the confusion. But resources seems to have images too. Should we have all of the images at one place under `content/images`

Reply:
You're correct.  I had missed that there are images in the book directories under 'resources'.  You're welcome to continue following that pattern.

Would you like me to revert my change to move the images into the `/images/` directory, or would you rather move them back yourself?

Reply:
Either is fine. Though in the longer run it would help categorizing what images should be in what folder (i.e. `content/images` OR `/images/`)